packages/create-vite/template-vue-ts/tsconfig.json

```diff
@@ -1,11 +1,25 @@
 {
-  "files": [],
-  "references": [
-    {
-      "path": "./tsconfig.app.json"
-    },
-    {
-      "path": "./tsconfig.node.json"
-    }
-  ]
+  "compilerOptions": {
+    "target": "ES2020",
+    "useDefineForClassFields": true,
+    "module": "ESNext",
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "skipLibCheck": true,
+
+    /* Bundler mode */
+    "moduleResolution": "bundler",
```

Comment:
you are missing `"moduleDetection": "force"`, this is important for any user that decides to use eslint within template, as it forces all files to be recognized as module's,

eg, vue files in setup mode will not be treated as such without this.

Reply:
Yeah I've only updated back the React one